src/apify_client/clients/resource_clients/log.py

```diff
@@ -175,3 +202,179 @@ async def stream(self) -> AsyncIterator[httpx.Response | None]:
         finally:
             if response:
                 await response.aclose()
+
+
+class StreamedLog:
+    """Utility class for streaming logs from another actor.
```

Comment:
Please use capitalized Actor (everywhere, there are many occurrences).

```suggestion
    """Utility class for streaming logs from another Actor.
```

Reply:
Done